PIL/ImageShow.py

```diff
@@ -69,8 +69,9 @@ def show(self, image, **options):
             # FIXME: auto-contrast if max() > 255?
         else:
             base = Image.getmodebase(image.mode)
-        if base != image.mode and image.mode != "1":
+        if base != image.mode and image.mode != "1" and image.mode != "RGBA":
             image = image.convert(base)
```

Comment:
I just tried saving with and without alpha like this to BMP:

```
>>> from PIL import Image
>>> a = Image.open("emoji-color-cairo.png")
>>> a.save("one.bmp")
>>> b = a.convert("RGB")
>>> b.save("two.bmp")
```
It results in two substantially different bmp in size:

```
-rw-rw-r--. 1 Hin-Tak Hin-Tak 204854 May 10 20:35 one.bmp
-rw-rw-r--. 1 Hin-Tak Hin-Tak 153654 May 10 20:35 two.bmp
```

xv shows them to be visually the same, but imagemagick's `display` shows the former with alpha like the origi. So BMP + alpha is, as you said, works on some viewer but not others. On windows, according to `PIL/ImageShow.py` line 108 (a few lines above my first change, which switches for darwin and unix), it launches `start "Pillow" /WAIT file`. This seems to be some sort of built-in viewer in Pillow?